numpy/linalg/linalg.py

```diff
@@ -2583,18 +2583,21 @@ def multi_dot(arrays): return functools.reduce(np.dot, arrays)
 
     >>> from numpy.linalg import multi_dot
     >>> # Prepare some data
-    >>> A = np.random.random(10000, 100)
-    >>> B = np.random.random(100, 1000)
-    >>> C = np.random.random(1000, 5)
-    >>> D = np.random.random(5, 333)
+    >>> A = np.random.random((10000, 100))
+    >>> B = np.random.random((100, 1000))
+    >>> C = np.random.random((1000, 5))
+    >>> D = np.random.random((5, 333))
     >>> # the actual dot multiplication
-    >>> multi_dot([A, B, C, D])
+    >>> _ = multi_dot([A, B, C, D])
+    ...
```

Comment:
I don't think you need both the `...` and the `_ =` - either should work alone

Reply:
True